src/main/java/com/urlparser/entity/LinksFromUrl.java

```diff
@@ -0,0 +1,55 @@
+package com.urlparser.entity;
+
+import com.urlparser.model.Links;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class LinksFromUrl implements Links {
+
+  private static String url;
```

Comment:
@undsett this should be instance field, not static, since we may want to create number of instances of `LinksFromUrl`, right?